Automattic/jetpack
# My Jetpack: Add wpcom plan slug to product info

#### Changes proposed in this Pull Request:

* Add the wpcom product slug to product info

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Visit My Jetpack
* Inspect the `myJetpackInitialState` global var
* Verify that every product has a `wpcom_product_slug` key
